modules/cluster-logging-collector-log-forward-update.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-external.adoc
+
+// Might not be needed. See https://issues.redhat.com/browse/LOG-654
+
+[id="cluster-logging-collector-log-forward-update_{context}"]
+= Updating log forwarding custom resources
+
+After updating to the Log Forwarding API GA, you will need to modify or recreate any existing Log Forwarding CRs that you have.
+
+If you edit your existing PRs, you must make the following modifications:
+
+.Updated Log Forwarding CR
+[sourcce,yaml]
+----
+apiVersion: "logging.openshift.io/v1" <1>
+kind: "ClusterLogForwarder" <2>
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  disableDefaultForwarding: true
+  outputs:
+   - name: elasticsearch
+     type: "elasticsearch"
+     endpoint: elasticsearch.openshift-logging.svc:9200 <3>
+  pipelines:
+   - name: container-logs
+      inputRefs: <4>
+        - infrastructure
+        - application   
+        - audit
+     outputRefs:
+     - elasticsearch
+     - secureforward-offcluster
+----
+
+<1> Change the API from `apiVersion: "logging.openshift.io/v1alpha1"` to `apiVersion: logging.openshift.io/v1`.
+
+<2> Change the object kind from `kind: "LogForwarding"` to `kind: ClusterLogForwarder`.
+
+<3> Change the output parameter from `spec.outputs.endpoint` to `spec.output.url`
```

Comment:
Look at the above example, some fields are changed to the new API, some fields aren't, could we maintain a uniform format ?